DESIGN_DOCS.md

```diff
@@ -0,0 +1,65 @@
+# Design Documents
+
+## Purposes of this document
+* To describe what our design documents are for, their structure, and what goes in them.
+* To describe how DSNP Design Documents are produced and where they're published.
+
+## What Is a Design Document?
+It's a document clearly describing a problem, a solution to that problem, and why we think the proposal is the right solution.
+It has enough information to begin writing stories for implementing the solution in the document.
+ 
+## Audience
+The audience for DSNP Design Documents is mainly DSNP developers, both internal and at large.
```

Comment:
Let's rephrase as "mainly DSNP developers, from multiple organizations".  (I.e., design docs should not presume knowledge that is internal to any one organization -- maybe what I'm saying here in parenthesis should actually go in the text, even?)

Reply:
I'm not sure how the rephrase substantially changes the meaning.

Reply:
Oh, it doesn't change the meaning; it changes the framing.  This document's audience is, ultimately, a broad developer community -- one that, assuming we succeed, will include multiple organizations, and one whose long-term governance is not yet known.  In that scenario, what do the descriptors "internal" and "at large" mean?  If "internal" means "at Unfinished", then... well, you see my point? :-)

Reply:
(I guess, now that I think about it, the suggested edit *does* change the meaning, by removing an implied meaning that we don't want to include.)